getting_started/administrators.adoc

```diff
@@ -31,63 +34,45 @@ your hosts, which includes verifying system and environment requirements and
 installing and configuring Docker. After ensuring your hosts are properly set
 up, you can continue by choosing one of the following installation methods.
 
+Docker and OpenShift must run on the Linux operating system. If you wish to
+run the server from a Windows or Mac OS X host, you should download and run
+the Origin Vagrant image as described in link:#building-from-source[Method 3].
+
 == Installation Methods
 
 Choose one of the following installation methods that works best for you.
 
 === Method 1: Running in a Docker Container [[running-in-a-docker-container]]
 You can quickly get OpenShift running in a Docker container using images from
-https://hub.docker.com[Docker Hub].
+https://hub.docker.com[Docker Hub] on a Linux system.
 
 *Installing and Starting an All-in-One Server*
 
 . Launch the server in a Docker container:
 +
 ----
-$ docker run -d --name "openshift-origin" --net=host --privileged \
--v /var/run/docker.sock:/var/run/docker.sock \
--v /tmp/openshift:/tmp/openshift \
-openshift/origin start
+$ sudo docker run -d --name "origin" \
+        --privileged --net=host \
+        -v /:/rootfs:ro -v /var/run:/var/run:rw -v /sys:/sys:ro -v /var/lib/docker:/var/lib/docker:rw \
+        -v /var/lib/openshift/openshift.local.volumes:/var/lib/openshift/openshift.local.volumes \
```

Comment:
There have been issues in the past with at least the volumes dir not having the correct SELinux label (e.g. it gets `docker_var_lib_t` when it has to be `svirt_sandbox_file_t`). I am not sure if we need to change something in our node startup code that calls `chcon` or what, but it's probably worth another look to see if it's still a problem.